demos/yelp_demo/README.md

```diff
@@ -224,18 +158,118 @@ refreshes. Spawn a new pod with these resource requirements:
 | Memory                | 48G                                                                  |
 | Extra Resource Limits | `{"nvidia.com/gpu":2}`                                               |
 
+It will take a while for the pod to spawn. While you're waiting, watch for
+autoprovisioning to occur. View the Workload and Node status in the GCP console.
+
 Once the notebook environment is
 available, open a new terminal and upload this
 [Yelp notebook](notebooks/yelp.ipynb).
 
-Execute the notebook to show that it works.
+Ensure the kernel is set to Python 2, then execute the notebook.
+
+## 6. Run a simple pipeline
+
+Show the file `gpu-example-pipeline.py` as an example of a simple pipeline.
+
+Compile it to create a .tar.gz file:
+
+```
+./gpu-example-pipeline.py
+```
+
+View the pipelines UI locally by forwarding a port to the ml-pipeline-ui pod:
+
+```
+PIPELINES_POD=$(kubectl get po -l app=ml-pipeline-ui | \
```

Comment:
With the latest version of kubectl you can port-forward to a service
https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/

If you use a service its one command because services have stable names.

Reply:
oh that's much better!